frontend/awx/views/schedules/components/RulesPreview.tsx

```diff
@@ -10,80 +10,71 @@ import {
-import { WizardFormValues } from '../../../resources/templates/WorkflowVisualizer/types';
-import { ScheduleFormWizard } from '../types';
-import { RRule, RRuleSet, rrulestr } from 'rrule';
-import { formatDateString } from '../../../../../framework/utils/formatDateString';
-import { Options } from 'rrule';
+import { RRuleSet } from 'rrule';
+import { DateTime } from 'luxon';
 
-export function RulesPreview(props: { rrule?: string }) {
+export function RulesPreview(props: { ruleSet?: RRuleSet }) {
   const { t } = useTranslation();
-  const { wizardData } = usePageWizard() as {
-    wizardData: WizardFormValues & ScheduleFormWizard;
-    visibleSteps: PageWizardStep[];
-  };
+  const ruleSet = props.ruleSet ?? new RRuleSet();
 
   return (
     <PageFormSection singleColumn title={t('Preview of the first 10 rules')} canCollapse>
-      {!props.rrule ? (
-        wizardData.rules?.map((element, index) => {
-          return <RulesPreviewBody ruleObject={element.rule.options} key={index} index={index} />;
-        })
-      ) : (
-        <RulesPreviewBody ruleString={props.rrule} />
-      )}
+      <RulesPreviewBody ruleSet={ruleSet} />
     </PageFormSection>
   );
 }
 
-export function RulesPreviewBody(props: {
-  ruleString?: string;
-  ruleObject?: Options;
-  index?: number;
-}) {
+export function RulesPreviewBody(props: { ruleSet: RRuleSet }) {
   const { t } = useTranslation();
-  let newRule;
-  const rRuleArray = [] as RRule[];
-
-  if (props.ruleString) {
-    const rRruleString = rrulestr(props.ruleString, { forceset: true }) as RRuleSet;
-    rRruleString._rrule.map((value: RRule) => {
-      const rRule = value.options as Options;
-      newRule = new RRule(rRule);
-      rRuleArray.push(newRule);
-    });
-  } else {
-    newRule = new RRule(props.ruleObject);
-    rRuleArray.push(newRule);
-  }
 
   return (
     <>
-      {rRuleArray.map((rule, index) => {
+      {props.ruleSet.rrules().map((rule, index) => {
         return (
           <>
-            <Title headingLevel={'h4'}>
-              {props.index ? t(`Rule ${props.index + 1}`) : t(`Rule ${index + 1}`)}
-            </Title>
-            <DescriptionList
-              data-cy={props.index ? `rule-${props.index + 1}` : `rule-${index + 1}`}
-              isCompact
-            >
+            <Title headingLevel={'h4'}>{t(`Rule ${index + 1}`)}</Title>
+            <DescriptionList data-cy={`rule-${index + 1}`} isCompact>
               <DescriptionListGroup>
-                <DescriptionListTerm>{t('Rule preview')}</DescriptionListTerm>
+                <DescriptionListTerm>{t('Rule preview local')}</DescriptionListTerm>
                 <LabelGroup numLabels={5}>
                   {rule
                     .all((_rule, i) => i < 10)
                     ?.map((value, i) => {
-                      return <Label key={i}>{formatDateString(value)}</Label>;
+                      const localDateTime = DateTime.fromISO(value.toISOString())
```

Comment:
i think setting `setZone` to true is missing because times don't match in my machine
` DateTime.fromISO(value.toISOString(), { setZone: true })`